### PR TITLE
Add TS0601_gas_sensor_2

### PR DESCRIPTION
**Pull request by Koenkk**
See https://gist.github.com/Koenkk/86ea747a683752fd6ff9390abf9fda55

### CODE REVIEW

File: devices/tuya.js [hunk 1]
```diff
@@ -869,6 +869,37 @@ module.exports = [
...
+        fromZigbee: [tuya.fz.datapoints],
+        toZigbee: [tuya.tz.datapoints],
+        configure: tuya.configureMagicPacket,
+        exposes: [
+            e.gas(), tuya.exposes.gasValue().withUnit('LEL'), tuya.exposes.selfTest(), tuya.exposes.selfTestResult(),
+            tuya.exposes.silence(),
+            exposes.enum('alarm_ringtone', ea.STATE_SET, ['1', '2', '3', '4', '5']).withDescription('Ringtone of the alarm'),
+            exposes.numeric('alarm_time', ea.STATE_SET).withValueMin(1).withValueMax(180).withValueStep(1)
+                .withUnit('s').withDescription('Alarm time'),
+            exposes.binary('preheat', ea.STATE, true, false),
```
Comment (Koenkk) [inline devices/tuya.js:886]: @Masterz69  can you provide a description for this?

Comment (Masterz69) [reply devices/tuya.js:886]: 'Indicates sensor preheat is active'

File: devices/tuya.js [hunk 2]
```diff
@@ -869,6 +869,37 @@ module.exports = [
...
+            exposes.binary('alarm_switch', ea.STATE_SET, true, false),
```
[the remark this answers is not in the document]
Comment (Masterz69) [reply devices/tuya.js:887]: Actually at moment not sure of functionality of this. In Tuya Smart app it have name just 'Alarm Switch', but it not disabling switch on device. Probably later will discover what it does. May be leave 'Alam Switch' for now.

[the remark this answers is not in the document]
Comment (Koenkk) [reply devices/tuya.js:887]: Since we don't know what it does, lets ignore it for now.

**Pull request merged**